.github/workflows/run_tests.yml

```diff
@@ -25,6 +25,18 @@ jobs:
         with:
           version: latest
 
+  gomodcheck:
+    name: Check internal dependencies
```

Comment:
Do we need the same CircleCI job?

Reply:
We could but I see no reason to duplicate this, Github actions work as expected.

Reply:
Historically we have both, the presumption was that we'll choose between them based on real usage experience. But it looks like there are no significant differences to make a choice. Probably we're fine with Github only.